orga/tests/unit/services/session_test.js

```diff
@@ -215,8 +215,7 @@ module('Unit | Service | session', function (hooks) {
         service.handleLocale({ isFranceDomain, localeFromQueryParam, userLocale });
 
         // then
-        sinon.assert.notCalled(localeService.setLocaleCookie);
-        assert.ok(true);
+        assert.true(localeService.setLocaleCookie.notCalled);
```

Comment:
Pourrait-on réécrire ce message de commit en : 
"""
prefer assert.true(condition)
instead of 'sinon.assert.*' and 'assert.ok(true)'
"""